test/helper-window.cpp

```diff
@@ -173,6 +173,7 @@ static Fl_Double_Window* make_window() {
   { Fl_Double_Window* o = new Fl_Double_Window(1465, 860);
     w = o;
     { sim_win = new fltkview(10, 5, 1280, 720);
+      std::cout << " sim_win --------------------------------------------- \n";
```

Comment:
These look like debugging messages. Why commit them?

Reply:
Yes, this was a debugging message and was deleted in a later commit. I made this commit to save a checkpoint. Is it that no such debugging information should be committed? Even if it will be removed in the same PR?

Reply:
On projects I maintain, I expect PRs to consist of 'clean' series of commits. It's easy to get rid of ugly cruft with rebases, etc, while having extra material like this makes reading the important parts of diffs hard. 

Since I was asked to review, I did :)

Reply:
Got it! I will try to follow the standard. Thank you very much!

Reply:
I start a new PR #3  to get rid of the cruft.

Reply:
This branch is forced updated as well. Will close #3.